### DISCUSSION TITLE
WinUI3 controls look visually less appealing than their UWP counterparts

### DISCUSSION THREAD

**Discussion by Balkoth** [microsoft/WindowsAppSDK#1999] [General]
### Why weren't the styles just taken from UWP?
Maybe this is a personal thing, but to me the new styles just look way worse than whats in Microsoft.UI.Xaml.
Please share your experiences.

### Sample 1 - NavigationView

In UWP you can see the hovered item and the large underline clearly shows the current active item. Its dense, compact and crisp. The WinUI version does NOT show the hovered item. While you can't see the mouse cursor on the picture, i assure you that i hovered it while taking the screenshot. The small dotlike outline is hard to see.

UWP:
![uwp_01](https://user-images.githubusercontent.com/80055480/150514479-407bcebf-6f4f-4a49-b36f-2a9fe7ac3a7f.png)

WinUI3:
![winui3_01](https://user-images.githubusercontent.com/80055480/150514533-52fe0b22-0ea4-4634-a749-30189bdb5e32.png)

### Sample 2 - TabView
I don't think there is much to say about the UWP version, maybe that the vertical spacer is on top of the tab header. Otherwise it looks clear and well defined to me. I am speechless at how bad the WinUI3 version looks. I have not seen such a bad looking TabControl since owner drawn Windows XP tab controls. The button like selected tab doesn't even look like it belongs there. To me it looks like something glitched the rendering. The visual difference of the color between selected and hovered is so minimal that it is hard to tell which is which.

UWP:
![uwp_02](https://user-images.githubusercontent.com/80055480/150515402-0cd3a46f-cbfd-422c-a068-88eded2cccaf.png)

WinUI3:
![winui3_02](https://user-images.githubusercontent.com/80055480/150515426-e1482a12-56ec-42f3-af85-e03438b59d73.png)

**Comment by kmgallahan** [comment]
>  but to me the new styles just look way worse than whats in Microsoft.UI.Xaml

Microsoft.UI.XAML is WinUI 3

Windows.UI.XAML is UWP (WinUI 2)

IMO reveal, tilt, and liberal use of acrylic were just implemented to show off the composition engine. Using any of them on a regular basis just turns into a distraction and detracts from productivity.

New rounded corners get a 'meh' from me - at least there isn't a notch somewhere.

The new tabs are a mess at the moment. Not sure how much is attributable to bugs vs. design decisions.

**Reply by Balkoth** [reply]
I took the UWP name from the official github page 🤔. It is confusing even to experienced devs.

**Reply by kmgallahan** [reply]
Windows.* represents namespaces where the code is a part of the OS (UWP is built into Windows).

Microsoft.* was adopted for WinUI 3 since the code was lifted out of the OS for it.

WinUI 2 is just a rebranding of the UWP XAML UI framework.

Edit: For clarification the new styles contained in [Microsoft.UI.XAML](https://www.nuget.org/packages/Microsoft.UI.Xaml) are applied to the UWP XAML controls which live in the Windows namespace. The linked package is the only bit of code that lives outside of Windows and is what you use when building WinUI 2 apps.

**Reply by Balkoth** [reply]
Your edit implies there is a possibility to use the imho better looking style. How?

**Reply by kmgallahan** [reply]
This is what you get with WinUI 2: https://github.com/microsoft/microsoft-ui-xaml/tree/release/2.8.1cbs/dev

Some new controls that don't have old skins, some code for old controls that are now open source with new changes, and some skins that are applied to old controls (https://github.com/microsoft/microsoft-ui-xaml/tree/release/2.8.1cbs/dev/CommonStyles).

Some of the *_themeresources_v1.xaml skins in there might have the older look, but I've not tried using them.

If you want the old look, then you can either not use WinUI 2 and just use classic UWP, or try to use old skins by mixing them with WinUI 2: \(Program Files)\Windows Kits\10\DesignTime\CommonConfiguration\Neutral\UAP\<SDK version>\Generic

In short, there is not just a switch to flip to mix old skins with WinUI 2.
